srcpkgs/walk/template

```diff
@@ -0,0 +1,16 @@
+# Template file for 'walk'
+pkgname=walk
+version=1.0.0
+revision=1
+build_style=gnu-makefile
+hostmakedepends="gzip make"
+short_desc="Two utilities collectively replacing find: walk and sor"
+maintainer="Andrea D'Amore <a@d-amo.re>"
+license="Apache-2.0"
+homepage=https://git.benjamin.barenblat.name/walk/
```

Comment:
Add quotes

Reply:
How so?

`homepage` does not contain IFS, no accompanying file mentions having to quote it.

Reply:
It's not required, but almost every other template does it, so let's be consistent